server/controllers/finance/fiscal.js

```diff
@@ -738,12 +738,13 @@ function getPeriods(req, res, next) {
 
 /**
  * return a query for retrieving account'balance by type_id and periods
+ * In general in accounting the balance is obtained by making debit - credit
  */
 function getAccountBalancesByTypeId(rate = 1) {
   return `
     SELECT ac.id, ac.number, ac.label, ac.parent, (IFNULL(s.amount, 0)*${rate}) AS amount, s.type_id
     FROM account as ac LEFT JOIN (
-    SELECT SUM(pt.credit - pt.debit) as amount, pt.account_id, act.id as type_id
```

Comment:
Is this function used anywhere else?  We should make sure that this change doesn't affect any other reports.

Reply:
I made a request and I notice it only used in the Operating Account report \server\controllers\finance\reports\operating

Reply:
Super.  Looks good to me